src/System.Net.WebSockets.Client/tests/CancelTest.cs

```diff
@@ -117,23 +117,52 @@ public class CancelTest : ClientWebSocketTestBase
                 await cws.CloseOutputAsync(WebSocketCloseStatus.NormalClosure, "CancelShutdown", cts.Token);
             }, server);
         }
+        
+        [OuterLoop]
+        [ConditionalTheory(nameof(WebSocketsSupported)), MemberData(nameof(EchoServers))]
```

Comment:
Why is this a theory if it only takes a single input? 
Shouldn't it be a ConditionalFact instead?